### PR TITLE
Merge dev to master

### PR DESCRIPTION
**Pull request by jlaneve**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    };
+
+    @singleS3StreamOutput('json')
+    @singleS3StreamInput('json', true)
+    async onTask(activityTask, config) {
+        return this.transform(activityTask.input.file, config);
+    }
+
+    transform(stream, config) {
+        return stream.pipe(through2({ objectMode: true }, (chunk, encoding, callback) => {
```
Comment (trong-le) [inline lib/index.js]: Probable better to use the [through2-map library](https://www.npmjs.com/package/through2-map). Same thing minus the objectMode and extra needed parameters for the function so looks more clean

File: package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "aries-activity-github-source",
```
Comment (trong-le) [inline package.json]: Change to `aries-activity-filter-fields-transform`

File: package.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  "version": "1.0.0",
+  "description": "",
+  "main": "lib/index.js",
+  "scripts": {
+    "test": "node -r babel-register -r babel-polyfill -r dotenv/config test/test.js | faucet",
+    "tail": "tail -f app.log | bunyan -o short"
+  },
+  "author": "trong-le",
```
Comment (trong-le) [inline package.json]: change to your name

**Comment by trong-le** [comment]
also add circle.yml file

**Comment by jlaneve** [comment]
@trongle by that do you mean flattening the results by default?

**Comment by trong-le** [comment]
@jlaneve keeping it how it is but using a deep pick/deep omit so that users won't have to know use the flattened name to remove/pick the field

**Pull request merged**